### PR TITLE
Update ftp links to https

### PR DESCRIPTION
**Pull request by veidenberg**
This PR updates all `http://ftp` URLs to https (along with `ftp://` URLs that work with https).
The affected URL hostnames have been tested working with https.
Similar PRs in other repos are listed in the JIRA ticket.
Note: waiting for SSL cert update in ftp.ensemblgenoems.org

JIRA ticket: https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6632
Sandbox: http://wp-np2-1d.ebi.ac.uk:1680/

### CODE REVIEW

File: ensembl/htdocs/info/genome/compara/hmm_lib.html [hunk 1]
```diff
@@ -85,13 +85,13 @@ <h2 id="stats">Statistics</h2>
   <h2 id="references">Notes and References</h2>
 
   <ol>
-      <li id="fn1">PANTHER HMM scoring tools - Version 1.03. Can be downloaded <a href="http://ftp.pantherdb.org//hmm_scoring/current_release">here</a></li>
+      <li id="fn1">PANTHER HMM scoring tools - Version 1.03. Can be downloaded <a href="http://data.pantherdb.org/ftp/hmm_scoring/current_release/">here</a></li>
```
Comment (ens-ap5) [inline ensembl/htdocs/info/genome/compara/hmm_lib.html:88]: Is this correct? You haven't changed the http to https

**Review by ens-ap5 — COMMENTED** [review]
Mostly looks fine.

**Comment by jyothishnt** [comment]
Wait until SSL certificate is updated (ticket with helpdesk) for ftp.ensemblgenomes.org

**Comment by veidenberg** [comment]
Feedback from hellpdesk: the working alias for ensemblgenomes is https://ftp.ensemblgenomes.ebi.ac.uk
Code updated

File: covid19/conf/ini-files/DEFAULTS.ini [hunk 2]
```diff
@@ -8,7 +8,7 @@ DEFAULT_FAVOURITES       = [Sars_cov_2]
 ENSEMBL_BLOG_URL          = http://www.ensembl.info/category/02-covid-19/
 ENSEMBL_BLOG_RSS          = http://www.ensembl.info/feed/
 
-ENSEMBL_FTP_URL           = http://ftp.ensemblgenomes.org/pub/viruses
+ENSEMBL_FTP_URL           = https://https://ftp.ensemblgenomes.ebi.ac.uk/pub/viruses
```
Comment (azangru) [inline covid19/conf/ini-files/DEFAULTS.ini:11]: Too much https.

Comment (veidenberg) [reply covid19/conf/ini-files/DEFAULTS.ini:11]: whoops

**Comment by veidenberg** [comment]
Update (feedback from Andy): we are good to use https://ftp.ensemblgenomes.ebi.ac.uk alias for now, but would like to get the SSL certificate https://ftp.ensemblgenomes.org working. So I think we can merge this PR until that happens (ServiceNow ticket created).

**Pull request merged**